src/app/comment-period/submit-comment-modal/submit-comment-modal.spec.ts

```diff
@@ -20,8 +20,9 @@ import { Document } from '../../models/document';
 import { ValuedComponent } from '../../models/vcs';
```

Comment:
It looks like all this test file could be removed, since there are no meaningful tests that need to run.